CHANGELOG.rst

```diff
@@ -12,6 +12,11 @@ Change Log
 Unreleased
 ----------
 
```

Comment:
Shouldn't the updated version and date be up here?

Reply:
We add the date once we are ready to release. The next release is going to also include PR#155 so I added my changes to make it easier for whoever has to make the new release.